README.md

```diff
@@ -116,36 +110,58 @@ DIR=<some-path-that-will-contain-data-and-results>
+* `m3_data_design.py`: The data design that is used for validating the M3 model.
+
+In this example, we will generate data for the single publisher models and then analyze the
+performance of these models.  The next command invokes the synthetic data generator.  
+
 
 ```
-python3 data_generators/synthetic_data_generator.py \
-  -nreps=10 \
-  -npublishers=2 \
-  -cost_model=fixed \
-  -frequencies=heterogeneous:3:5 \
-  -prices=0.05,0.06,0.07 \
-  -overlap=independent
-  -output=$DIR/data_design
+python3 data_generators/synthetic_data_design_generator.py \
+  --output_dir=$DIR/data \
+  --data_design_config=data_generators/single_publisher_design.py
 ```
 
-In this example, we will use the sample experimental design in the file
-`examples/experimental_designs/goerg_gamma_poisson.py`. This compares [Georg Goerg's
-one-point model](https://research.google/pubs/pub43218/) to the Gamma-Poisson distribution,
-using the Pairwise Union model for overlaps.
+After running this command, you should see that the directory `$DIR/data` contains many
+subdirectories.  The directory `$DIR/data` is a Data Design, and each of the subdirectories
+within it represents a Dataset.  Each Dataset in turn contains a collection of files representing
+synthetic impression logs.  There is one such file per synthetic publisher.  These files are
+just CSV files, so you can view them.
+
+An Experimental Design specifies a collection of different models and parameters.  An Experiment
+consists of running each of these against every Dataset in a Data Design.  Several example
+Experimental Designs can be found in the `driver` subdirectory:
+
+* `sample_experimental_design.py`:  A very simple example.
+* `single_publisher_design.py`: An experimental design that compares Goerg's one point model
+against the Gamma-Poisson model in a variety of settings.
+* `m3_first_round_experimental_design.py`: An experimental design for evaluating the proposed
+M3 model.
+
+The following command will evaluate the Experiments defined in `driver/single_publisher_design.py`
+against the Datasets that were created in the previous step:
+
 
 ```
-python3 drivers/experimental_design_driver.py \
-  -experimental_design=examples/experimental_designs/goerg_gamma.py \
-  -data_design=$DIR/data_design \
-  -intermediate_results=$DIR/intermediate_results \
-  -output=$DIR/results
+python3 driver/experiment_driver.py \
+  --data_design_dir=$DIR/data \
+  --experimental_design=driver/single_publisher_design.py \
+  --output_file=$DIR/results \
+  --intermediates_dir=$DIR/intermediates \
+  --cores=0
 ```
 
-This will generate an analysis of the two models against the ten data sets,
-which will be recorded in the file `$DIR/results`. You can then see an analysis
-of this by using the colab, `analysis/two_pub_analysis.ipynb`.
+Setting `cores=0` enables multithreading.  Even so, the above design takes very long to run, so you
```

Comment:
may clarify that "enables multithreading using all available cores."

Reply:
Done.